instagram/src/styles/ProfileEditPage/ItemDetail_style.jsx

```diff
@@ -0,0 +1,43 @@
+import styled from "@emotion/styled";
+
+export const ItemBox = styled.div`
```

Comment:
```suggestion
export const ProfileEditItemBox = styled.div`
```
와 같이 좀 더 디테일하게 컴포넌트 및 변수 작명하는게 좋음

Reply:
수정했습니다. 피드백 감사합니다.